include/posix/pthread.h

```diff
@@ -383,4 +422,120 @@ int pthread_barrierattr_getpshared(const pthread_barrierattr_t *, int *);
 int pthread_barrierattr_setpshared(pthread_barrierattr_t *, int);
 */
 
+
+/* Base Pthread related APIs */
+
+/**
+ * @brief POSIX pthread attributes initialization
+ *
+ * See IEEE 1003.1
+ */
+static inline int pthread_attr_init(pthread_attr_t *attr)
+{
+	*attr = initpthreadattrs;
+	return 0;
+}
+
+/**
+ * @brief POSIX pthread attributes destroy
+ *
+ * See IEEE 1003.1
+ */
+static inline int pthread_attr_destroy(pthread_attr_t *attr)
+{
+	return 0;
+}
+
+/**
+ * @brief POSIX get pthread scheduling parameter
+ *
+ * See IEEE 1003.1
+ */
+static inline int pthread_attr_getschedparam(const pthread_attr_t *attr,
+		struct sched_param *schedparam)
+{
+	schedparam->priority = attr->priority;
+	return 0;
+}
+
+/**
+ * @brief POSIX get pthread stack inforamtion from thread attribute
+ *
+ * See IEEE 1003.1
+ */
+static inline int pthread_attr_getstack(const pthread_attr_t *attr,
+		void **stackaddr, size_t *stacksize)
```

Comment:
parameters alignment: should be under last '(' + one space.

Apply that everywhere